agent/taskresource/envFiles/envfile.go

```diff
@@ -32,34 +33,40 @@ import (
 
 	"github.com/aws/amazon-ecs-agent/agent/api/task/status"
 	"github.com/aws/amazon-ecs-agent/agent/taskresource"
+	"github.com/aws/amazon-ecs-agent/agent/utils/bufiowrapper"
 	"github.com/aws/amazon-ecs-agent/agent/utils/ioutilwrapper"
 )
 
 const (
-	ResourceName      = "envfile"
-	envFileDirPath    = "envfiles"
-	envTempFilePrefix = "tmp_env"
-	envFileExtension  = ".env"
+	ResourceName         = "envfile"
+	envFileDirPath       = "envfiles"
+	envTempFilePrefix    = "tmp_env"
+	envFileExtension     = ".env"
+	commentIndicator     = "#"
+	envVariableDelimiter = "="
 
 	s3DownloadTimeout = 30 * time.Second
 )
 
 // EnvironmentFileResource represents envfile as a task resource
 // these environment files are retrieved from s3
 type EnvironmentFileResource struct {
-	cluster     string
-	taskARN     string
-	region      string
-	resourceDir string // path to store env var files
+	cluster       string
+	taskARN       string
+	region        string
+	resourceDir   string // path to store env var files
+	containerName string
```

Comment:
i didn't see containerName and environmentFilesLocation added to environmentFileResourceJSON. they probably should?

Reply:
ah yeah, good catch. will add.